### PR TITLE
Add campaigns search field

### PR DESCRIPTION
**Pull request by xavierbarbosa**
### Added
- Search field for the campaigns to search for obr_name and epid
- Add types to OrgUnit, so we can category an OrgUnit type and filter by it

### CODE REVIEW

File: plugins/polio/js/src/components/Dashboard.js [hunk 1]
```diff
@@ -909,6 +942,10 @@ export const Dashboard = () => {
         openCreateEditDialog();
     };
 
+    const handleSearch = useCallback(event => {
+        setSearchQuery(event.target.value);
```
Comment (kaiobrito) [inline plugins/polio/js/src/components/Dashboard.js:946]: I think It'd be nice to debounce this request to prevent overhead on the server

**Review by kaiobrito — APPROVED** [review]

File: plugins/polio/js/src/components/Dashboard.js (hunk 1 above)
Comment (xavierbarbosa) [reply plugins/polio/js/src/components/Dashboard.js:948]: done https://github.com/BLSQ/iaso/pull/211/commits/447aeaf0d98771eeaf7110a286e4240fb137273f

File: scripts/enable_plugins.sh [hunk 2]
```diff
@@ -1,5 +1,7 @@
 if [ "$PLUGIN_POLIO_ENABLED" == 'true' ]; then
     cd plugins/polio/js
+    node --version && npm --version
+    which npm
```
Comment (madewulf) [inline scripts/enable_plugins.sh:4]: This doesn't seem necessary, but I guess it won't break anything

File: iaso/migrations/0099_orgunittype_type.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    dependencies = [
+        ("iaso", "0097_add_account_feature_flag_catchment_20210624_0808"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="orgunittype",
+            name="type",
+            field=models.CharField(
+                blank=True, choices=[("COUNTRY", "Country"), ("DISTRICT", "District")], max_length=8, null=True
```
Comment (madewulf) [inline iaso/migrations/0099_orgunittype_type.py:17]: You added a type field on the org unit ... type ?

Comment (xavierbarbosa) [reply iaso/migrations/0099_orgunittype_type.py:17]: Yes after discussing with Kaio, this feels like a better approach than using an env variable. Felt like a very natural way of fixing the problem. We can call if you want to discuss it.

Comment (xavierbarbosa) [reply iaso/migrations/0099_orgunittype_type.py]: While chatting with @madewulf we decided to rename this field to category.

**Review by kaiobrito — APPROVED** [review]

**Pull request merged**